sdk/keyvault/keyvault-secrets/tests/secrets.test.ts

```diff
@@ -12,7 +12,7 @@ describe("Secret client", () => {
   const clientId = process.env["AAD_CLIENT_ID"] || "";
   const clientSecret = process.env["AAD_CLIENT_SECRET"] || "";
   const tenantId = process.env["AAD_TENANT_ID"] || "";
-  const vaultName = process.env["KEYVAULT_NAME"] || "<keyvault-name>";
+  const vaultName = (process.env["KEYVAULT_NAME"] || "<keyvault-name>").trim();
```

Comment:
Well now that we know the problem, we can probably revert this change. The ENV variable value has been properly trimmed in the datastore.